src/main/java/bio/terra/cli/businessobject/resource/GcsBucket.java

```diff
@@ -144,6 +148,18 @@ public String resolve(boolean includePrefix) {
     return includePrefix ? GCS_BUCKET_URL_PREFIX + bucketName : bucketName;
   }
 
+  /** Query the cloud for information about the bucket. */
+  public Optional<BucketCow> getBucket() {
```

Comment:
If this is just returning information about the bucket, IMO this should return an `Optional<BucketInfo>` instead of a BucketCow. If there are other uses for it though, BucketCow is fine.

Reply:
For this PR, we're only using information from the wrapped `BucketInfo` object, but I think in the future we want to also use the `BucketCow.list()` method to get the number of blobs in the bucket to include in the are-you-sure-you-want-to-delete-this prompt. So planning to leave this for now, but will keep in mind as I tackle the delete prompt ticket next.